app/Console/Commands/AdsProcessTx.php

```diff
@@ -0,0 +1,198 @@
+     */
+    const EXIT_CODE_CANNOT_GET_BLOCK_IDS = 1;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     * @param AdsClient $adsClient
+     * @return int
+     */
+    public function handle(AdsClient $adsClient)
+    {
+        try {
+            $this->updateBlockIds($adsClient);
+        } catch (CommandException $exc) {
+            $code = $exc->getCode();
+            $message = $exc->getMessage();
+            $this->error("Cannot update blocks due to CommandException:\n"
+                . "Code:\n  ${code}\n"
+                . "Message:\n  ${message}\n");
+            return self::EXIT_CODE_CANNOT_GET_BLOCK_IDS;
+        }
+
+        $adServerAddress = config('app.adshares_address');
+        $txs = AdsTxIn::where('status', AdsTxIn::STATUS_NEW)->get();
+        /** @var $tx AdsTxIn */
+        foreach ($txs as $tx) {
+            $transaction = $adsClient->getTransaction($tx->txid)->getTxn();
+            $type = $transaction->getType();
+            switch ($type) {
+                case 'send_many':
+                    $isTxTargetValid = false;
+                    /** @var $sendManyTx SendManyTransaction */
+                    $sendManyTx = $transaction;
+//                    $senderAddr = $sendManyTx->getSenderAddress();
+                    $wiresCount = $sendManyTx->getWireCount();
+                    if ($wiresCount > 0) {
+                        $wires = $sendManyTx->getWires();
+                        foreach ($wires as $wire) {
+                            /** @var $wire SendManyTransactionWire */
+                            $targetAddr = $wire->getTargetAddress();
+                            if ($targetAddr === $adServerAddress) {
+//                                $amount = $wire->getAmount();
+                                $isTxTargetValid = true;
+                                break;
+                            }
+                        }
+                    }
+                    $tx->status = $isTxTargetValid ? AdsTxIn::STATUS_RESERVED : AdsTxIn::STATUS_INVALID;
+                    $tx->save();
+                    break;
+
+                case 'send_one':
+                    /** @var $sendOneTx SendOneTransaction */
+                    $sendOneTx = $transaction;
+                    $targetAddr = $sendOneTx->getTargetAddress();
+                    if ($targetAddr === $adServerAddress) {
+                        $message = $sendOneTx->getMessage();
+                        $user = User::where('uuid', hex2bin($this->extractUuidFromMessage($message)))->first();
+                        if (null === $user) {
+                            $tx->status = AdsTxIn::STATUS_RESERVED;
+                            $tx->save();
+                        } else {
+//                            $senderAddr = $sendOneTx->getSenderAddress();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)